app/build.gradle.kts

```diff
@@ -13,9 +13,7 @@ dependencies {
         exclude(group = "com.exactpro.th2", module = "cradle-cassandra")
     }
 
-    implementation(platform(libs.jetty.bom)) {
-        because("vulnerabilities in version 11.0.13. Can be removed after updating Javalin and Kotlin")
-    }
+    implementation(platform(libs.jetty.bom))
```

Comment:
note: I don't think we need the Jetty bom anymore. It was added only to fix vulnerabilities in old Javalin version

Reply:
@OptimumCode  Do you mean we can remove all javalin dependencies exclude javalin openapi ?